### DISCUSSION TITLE
SEO optimisation

### DISCUSSION THREAD

**Discussion by rokas-bendikas** [alshedivat/al-folio#377] [Q&A]
Hi, is some sort of SEO provided in the package? If so, what and how to use it?

**Comment by rohandebsarkar** [comment]
The sitemap is generated by using the `jekyll-sitemap` plugin. And the theme also generates a `robots.txt` file at the root of the site with all access settings (you can modify it, if you want: [robots.txt](https://github.com/alshedivat/al-folio/blob/04f526db5d303aa3ce8c5e62089b7b6a0862f6a9/robots.txt)).

Apart from these, other SEO settings are not yet included in the theme.
